test/demarcheSimplifiees/autoVerifyPsychologist.spec.js

```diff
@@ -71,5 +71,5 @@ describe('autoVerifyPsychologist', () => {
           instructeurId: INSTRUCTOR_ID,
         },
       }));
-  }).timeout(4000);
+  }).timeout(30000);
```

Comment:
:o that's a lot.. should we ask demarches simplifiees about why is it that long the first time?

Reply:
i wanted to make sure we will never be bother by it :)